docs/FAQ.md

```diff
@@ -33,6 +33,41 @@ revision visible again.
 
 See [revsets] and [templates] for further guidance.
 
+### What are elided revisions in the output of `jj log`? How can I display them?
+
+"Elided revisions" appears in the log when one revision descends from another,
+both are in the revset, but the revisions connecting them are _not_ in the
+revset.
+
+For example, suppose you log the revset `tyl|mus` which contains exactly two
+revisions:
+
+```sh
+$  jj log -r 'tyl|mus'
+○  musnqzvt me@example.com 1 minute ago 9a09f8a5
+│  Revision C
+~  (elided revisions)
+○  tylynnzk me@example.com 1 minute ago f26967c8
+│  Revision A
+```
+
+Only the two revisions in the revset are displayed. "Elided revisions" is shown
+to indicate that `musnqzvt` descends from `tylynnzk`, but the nodes connecting
+them are not in the revset.
+
+To view the elided revisions, change the [revset expression](./revsets.md) so it
+includes the connecting revisions.  One option is `tyl::mus`:
```

Comment:
Nit: It may also be valuable to mention `ty..mus` as another example.

Reply:
That would then exclude `tyl`, so the output would change in _two_ ways. I think that would be distracting.

However, maybe I should change it to `connected(tyl|mus)`, because it is the same expression as before, just wrapped in something that un-elides anything that was elided.